### PR TITLE
PLAT-9262 handle thumb and volume mapped with encryption at rest

### PR DESCRIPTION
**Pull request by gotlieb**
@erankor 
@MosheMaorKaltura

### CODE REVIEW

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 1]
```diff
@@ -1058,12 +1073,26 @@ private static function captureMappedThumbUsingPackager($entry, $capturedThumbPa
...
 			return false;
 
-		$flavorUrl = myPlaylistUtils::buildPlaylistThumbPath($entry, $flavorAsset);
+		$flavorUrl = self::buildThumbUrl($entry, $flavorAsset);
 
 		return self::curlThumbUrlWithOffset($flavorUrl, $calc_vid_sec, $packagerCaptureUrl, $capturedThumbPath, $width, $height);
 	}
 
 
+	private static function buildThumbUrl($entry, $flavorAsset)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Move to protected (according to new Coding Standard)

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 2]
```diff
@@ -1142,16 +1171,12 @@ public static function captureLocalThumb($entry, $capturedThumbPath, $calc_vid_s
 		return true;
 	}
 
-	public static function captureLocalThumbUsingPackager($entry, $capturedThumbPath, $calc_vid_sec, &$flavorAssetId, $width, $height)
+	public static function captureLocalThumbUsingPackager($flavorAsset, $capturedThumbPath, $calc_vid_sec, &$flavorAssetId, $width, $height)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Either validate that flavorAsset is valid , or move the function to protected.

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 3]
```diff
@@ -2122,6 +2145,33 @@ public static function getVolumeMapContent($flavorAsset)
...
+			return self::retrieveMappedVolumeMapFromPackager($flavorAsset);
+		}
+		return self::retrieveLocalVolumeMapFromPackager($flavorAsset);
+	}
+
+	private static function retrieveMappedVolumeMapFromPackager($flavorAsset)
+	{
+		$packagerVolumeMapUrlPattern = kConf::get('packager_mapped_volume_map_url', 'local', null);
+		if (!$packagerVolumeMapUrlPattern)
+			throw new KalturaAPIException(KalturaErrors::VOLUME_MAP_NOT_CONFIGURED);
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Add brackets even on single line

Comment (gotlieb) [reply alpha/apps/kaltura/lib/myEntryUtils.class.php]: done

**Pull request merged**